gcode.c

```diff
@@ -726,6 +758,14 @@ char *gc_normalize_block (char *block, char **message)
     if(*block == '/')
         block++;
 
+#if STRING_REGISTERS_ENABLE
+    // If the block starts with '@' it means it is setting a string register value,
+    // and we should not normalize it
+    if(*block == '@') {
+        return block;
+    }
+#endif
```

Comment:
One thing I was considering is to use quotes for strings, and do something similar to comments, where characters between quotes are not modified.

So instead of `@5=This is a test`, you would write `@5="This is a test"`. This might help with one issue I had with expressions, where putting a space between values and operators causes an invalid expression error ( so `@[#<someParam> + 5]=Test` gives error, but `@[#<someParam>+5]=Test` does not. In my opinion spaces should just be ignored, and I think that is done by the block normalization? 🤔